physics/module_MYNNPBL_wrapper.F90

```diff
@@ -182,7 +183,7 @@ SUBROUTINE mynnedmf_wrapper_run(        &
      !logical, intent(in) :: mix_chem, fire_turb
      !integer, intent(in) :: nchem, ndvel, kdvel
      !for testing only:
-     logical, parameter :: mix_chem=.false., fire_turb=.false.
+     !logical, parameter :: mix_chem=.false., fire_turb=.false.
      integer, parameter :: nchem=2, ndvel=2, kdvel=1
 
```

Comment:
Can these parameters (nchem, vdvel, and kdvel) be input into the scheme? That is, they are probably determined by whoever is creating the smoke application, so they are probably determined outside of the MYNN. If they are changed in the future, it would be best to have them flexibly determined by the user rather than being hard-coded here. They were only hard-coded here for testing.

Reply:
In the updated PR, nchem and ndvel are input into the MYNN scheme. We just keep the parameter of kdvel=1 in the scheme, to be consistent with WRF-Chem.